sdk/keyvault/azure-keyvault-certificates/tests/test_certificates_client.py

```diff
@@ -387,7 +387,7 @@ def test_recover_and_purge(self, vault_client, **kwargs):
             client.purge_deleted_certificate(name=certificate_name)
 
         if not self.is_playback():
-            time.sleep(30)
+            time.sleep(50)
```

Comment:
We have [polling methods](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/keyvault/azure-keyvault-keys/tests/test_key_client.py#L336) to avoid this sort of thing.